build.gradle

```diff
@@ -132,7 +139,9 @@ task gitRevList(type: Exec) {
   standardOutput = new ByteArrayOutputStream()
 
   doLast {
-    new File('build', 'revision.txt').text = standardOutput.toString().trim()
+    new File('build/revision.txt').text = standardOutput.toString().trim()
```

Comment:
Ostensibly the two-arg constructor we were previously using was ever so slightly more portable, but I don't feel strongly about it.

We also weren't being consistent, which we now are.